src/services/useSaveIntegration.ts

```diff
@@ -33,14 +32,14 @@ export const saveIntegrationActionCreator = (
 
 const decoder = (
   response:
-    | Operations.EndpointResourceCreateEndpoint.Payload
-    | Operations.EndpointResourceUpdateEndpoint.Payload
+    | Operations.EndpointResource$v1CreateEndpoint.Payload
+    | Operations.EndpointResource$v1UpdateEndpoint.Payload
 ) => {
   if (response.type === 'Endpoint') {
     return {
       ...response,
       type: 'Integration',
-      value: toIntegration(response.value),
+      value: toIntegration(response.value as Schemas.EndpointDTO),
```

Comment:
We could use the Schemas.Endpoint instead
```suggestion
      value: toIntegration(response.value as Schemas.Endpoint),
```